view/frontend/web/js/productlabel-position.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this module to newer
+ * versions in the future.
+ *
+ * @category  Smile
+ * @package   Smile\ProductLabel
+ * @author    Houda EL RHOZLANE <houda.elrhozlane@smile.fr>
+ * @copyright 2019 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+
+define([
+        "jquery"
+    ], function($){
+
+        var widthOfParent = $("#labels").parent().width();
+        var heightOfParent = $("#labels").parent().height() - 44;
```

Comment:
This is also causing a strange behavior, on product listing, the panel goes "over" the product image and the product name, preventing from clicking on the image/name of the product : 

![Sélection_653](https://user-images.githubusercontent.com/15340849/55077109-3fab8580-5097-11e9-8eef-0fd546c29626.png)

Reply:
Done